rbac/internal/views.py

```diff
@@ -235,6 +240,97 @@ def sync_schemas(request):
     return HttpResponse('Invalid method, only "POST" is allowed.', status=405)
 
 
+@bop_request_time_tracking.time()
+def get_org_admin(request, org_or_account):
+    """Get the org admin for an account.
+
+    GET /_private/api/utils/get_org_admin/{org_or_account}/?type=account_id,org_id
+    """
+    PROXY = PrincipalProxy()
+    default_limit = StandardResultsSetPagination.default_limit
+    request_path = request.path
+    try:
+        limit = int(request.GET.get("limit", default_limit))
+        offset = int(request.GET.get("offset", 0))
+    except ValueError:
+        error = {
+            "detail": "Values for limit and offset must be positive numbers.",
+            "source": "get_org_admin",
+            "status": str(status.HTTP_400_BAD_REQUEST),
+        }
+        errors = {"errors": [error]}
+        return HttpResponse(errors, status=status.HTTP_400_BAD_REQUEST)
+
+    previous_offset = 0
+    if offset - limit > 0:
+        previous_offset = offset - limit
+    if request.method == "GET":
+        option_key = "type"
+        valid_values = ["account_id", "org_id"]
+        api_type_param = request.GET.get(option_key)
+        if api_type_param:
```

Comment:
little nit pick for readability: I think it would be a bit easier to follow if we inverted the if/else here to check for the absence of `option_key`, that way the corresponding `else` to this `if` isn't a long way down

so like 
`if !api_type_param: 
    return HttpResponse(...)
 ... rest of request continues`

Reply:
👍 Agreed! I pushed this change up 😁